test/es-module/test-esm-double-encoding-native%20.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
+require('../common');
+module.exports = 42;
```

Comment:
Could you add a comment describing this test. Maybe something like:
```js
'use strict';
require('../common');

// Trivial test to assert we can load files with `%` in their pathname.
// Also used by `test-esm-double-encoding.mjs`.

module.exports = 42;
```
As per https://github.com/nodejs/node/blob/master/doc/guides/writing-tests.md#test-structure